crates/table/src/layout.rs

```diff
@@ -22,12 +22,16 @@ use spacetimedb_sats::{
 ///
 /// When `base` is already aligned, `base` will be returned.
 pub const fn align_to(base: usize, required_alignment: usize) -> usize {
-    let misalignment = base % required_alignment;
-    if misalignment == 0 {
+    if required_alignment == 0 {
         base
     } else {
-        let padding = required_alignment - misalignment;
-        base + padding
+        let misalignment = base % required_alignment;
+        if misalignment == 0 {
+            base
+        } else {
+            let padding = required_alignment - misalignment;
+            base + padding
+        }
```

Comment:
This was panicking in my test case for `AlgebraicType::never()`, so I added a branch to avoid division by zero.